spec/factories/tasks.rb

```diff
@@ -5,5 +5,10 @@
     from = Date.parse("2019/08/01")
     to   = Date.parse("2019/12/31")
     deadline { Random.rand(from..to) }
+
+    trait :done_yesterday do
```

Comment:
yesterdayであることに特にこだわりは無いので doneでも良い気がします。
```suggestion
    trait :done do
```

Reply:
1750308 で修正を確認。